PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -355,6 +366,72 @@ RUN=`opts_GetOpt1 "--printcom" $@`
 UseJacobian="$(echo ${UseJacobian} | tr '[:upper:]' '[:lower:]')"
 UseJacobian=`opts_DefaultOpt $UseJacobian "true"`
 
+
+# ------------------------------------------------------------------------------
+#  Check MMP Version
+# ------------------------------------------------------------------------------
+
+Compliance="hcp"
+MPPVersion=`opts_GetOpt1 "--mppversion" $@`
+MPPVersion=`opts_DefaultOpt $MPPVersion "hcp"`
+
+if [ "${MPPVersion}" = "legacy" ] ; then
+  log_Msg "Legacy Minimal Preprocessing Pipelines: PreFreeSurferPipeline v.XX"
+  log_Msg "NOTICE: You are using MPP version that enables processing of images that do not"
+  log_Msg "        conform to the HCP specification as described in Glasser et al. (2013)!"
+  log_Msg "        Be aware that if the HCP requirements are not met, the level of data quality"
+  log_Msg "        can not be guaranteed and the Glasser et al. (2013) paper should not be used"
+  log_Msg "        in support of this workflow. A mnauscript with comprehensive evaluation for"
+  log_Msg "        the Legacy MPP workflow is in active preparation and should be appropriately"
+  log_Msg "        cited when published."
+  log_Msg "        "
+  log_Msg "Checking available data and processing options"
+else
+  log_Msg "HCP Minimal Preprocessing Pipelines: PreFreeSurferPipeline v.XX"
+  log_Msg "Checking data compliance with HCP MPP"
+fi
+
+# -- T2w image
+
+if [ "${T2wInputImages}" = "NONE" ] ; then
+  log_Msg "-> T2w image not present (legacy)"
+  SkipT2wImage="YES"
+  Compliance="legacy"
+else
+  SkipT2wImage="NO"
+  log_Msg "-> T2w image present (HCP)"
+fi
+
+# -- Use of custom brain
+
+if [ ! "${CustomBrain}" = "NONE" ] ; then
+  log_Msg "-> Custom brain used: ${CustomBrain} (legacy)"
+  Compliance="legacy"
+else
+  log_Msg "-> No custom brain used (HCP)"
+fi
+
+# -- Final evaluation
+
+if [ "${MPPVersion}" = "legacy" ] ; then
+  if [ "${Compliance}" = "legacy" ] ; then
+    log_Msg "Processing will continue using Legacy MPP."
+  else
+    log_Msg "All conditions for the use of HCP MPP are met. Consider using HCP MPP instead of Legacy MPP."
```

Comment:
Use of `log_Warn` seems more appropriate here.

Reply:
Changed to `log_Warn`